circle.yml

```diff
@@ -149,20 +149,33 @@ commands:
         default: 477
 
     steps:
+      - run:
+          name: "Install WABT"
+          command: |
+            if type wast2json; then
+              wast2json --version
+            else
+              [[ $OSTYPE = darwin* ]] && os=macos || os=ubuntu
+              cd /usr/local
+              curl -L https://github.com/WebAssembly/wabt/releases/download/1.0.19/wabt-1.0.19-$os.tar.gz | sudo tar xz --strip 1
+            fi
       - run:
           name: "Download spectest files"
           working_directory: ~/build
           command: |
             if [ ! -d wasm-spec ]; then
-              git clone https://github.com/wasmx/wasm-spec --branch w3c-1.0-jsontests-20200813 --depth 1
+              git clone https://github.com/wasmx/wasm-spec --branch w3c-1.0-jsontests-20200813 --depth 1 wasm-spec
```

Comment:
Without updating this tag, the benefit of this PR is the scaffolding is there, we can change the tag anytime. Is that correct? Or do you plan to change the tag in this PR?

Reply:
It may be better to merge it as is.

I'd like to create the 1.0 branch with test changes backported, but we have not decided yet what should be the base of this branch.

Reply:
Should we update `test/fizzy-spectests/README` in this PR?

Reply:
Still waiting for an answer here.

Reply:
Done.